### PR TITLE
:sparkles: Add http forwarding section

### PR DESCRIPTION
**Pull request by sivachandran**
## Why

http log forwarding support will be added

## What's changed

Add information about http log forwarding

### CODE REVIEW

**Review by chadwcarlson — DISMISSED** [review]

File: docs/src/increase-observability/logs/forward-logs.md [hunk 1]
```diff
@@ -180,10 +180,10 @@ You can use HTTP forwarding to forward Platform.sh logs to such third-party serv
 
 HTTP forwarding makes a `POST` HTTP request with an `application/json` body while forwarding the log messages to the endpoint.
 
-To enable HTTP forwarding, run a command similar to the following:
+To forward logs to Elasticsearch/OpenSearch using HTTP log forwarding, run the following command:
```
Comment (chadwcarlson) [inline docs/src/increase-observability/logs/forward-logs.md:183]: > As an example, to forward logs to Elasticsearch/OpenSearch using HTTP log forwarding, run the following command:

File: docs/src/increase-observability/logs/forward-logs.md [hunk 2]
```diff
@@ -180,10 +180,10 @@ You can use HTTP forwarding to forward Platform.sh logs to such third-party serv
...
 
 ```
-platform integration:add --type httplog --url "https://{{< variable "THIRD_PARTY_SERVICE_URL" >}}/{{< variable "INDEX_NAME" >}}/_docs" --headers '{ "Authorization": "Basic <basic_auth_token>", "Content-Type":"application/json" }'
+platform integration:add --type httplog --url "https://{{< variable "ES_OS_URL" >}}/{{< variable "INDEX_NAME" >}}/_doc" --header "Authorization: Basic <basic_auth_token>" --header "Content-Type: application/json"
```
Comment (chadwcarlson) [inline docs/src/increase-observability/logs/forward-logs.md:186]:
```sh
platform integration:add --type httplog --url "https://{{< variable "ELASTICSEARCH_URL" >}}/{{< variable "INDEX_NAME" >}}/_doc" --header "Authorization: Basic <basic_auth_token>" --header "Content-Type: application/json"

```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 3]
```diff
@@ -172,6 +172,26 @@ follow these steps:
...
 
+## Forward to an HTTP endpoint
+
+Some third-party services, such as [Elasticsearch](../../add-services/elasticsearch.md) and [OpenSearch](../../add-services/opensearch.md),
+support ingesting log messages through an HTTP endpoint.
+You can use HTTP forwarding to forward Platform.sh logs to such third-party services.
+
+HTTP forwarding makes a `POST` HTTP request with an `application/json` body while forwarding the log messages to the endpoint.
+
+To forward logs to Elasticsearch/OpenSearch using HTTP log forwarding, run the following command:
```
Comment (AnouckColson) [inline docs/src/increase-observability/logs/forward-logs.md:183]:
```suggestion
As an example, to forward logs to Elasticsearch using HTTP log forwarding, run the following command:
```

File: docs/src/increase-observability/logs/forward-logs.md [hunk 4]
```diff
@@ -172,6 +172,26 @@ follow these steps:
...
+
+```
+platform integration:add --type httplog --url "https://{{< variable "ES_OS_URL" >}}/{{< variable "INDEX_NAME" >}}/_doc" --header "Authorization: Basic <basic_auth_token>" --header "Content-Type: application/json"
```
Comment (AnouckColson) [inline docs/src/increase-observability/logs/forward-logs.md:186]:
```suggestion
platform integration:add --type httplog --url "https://{{< variable "ELASTICSEARCH_URL" >}}/{{< variable "INDEX_NAME" >}}/_doc" --header "Authorization: Basic <basic_auth_token>" --header "Content-Type: application/json"
```

**Review by AnouckColson — APPROVED** [review]

**Pull request merged**